### ISSUE TITLE
The export all file does not pull through Club field

### ISSUE THREAD

**Issue by GoogleCodeExporter** [geoffhumphrey/brewcompetitiononlineentry#458]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


Please provide any additional information below.


```

Original issue reported on code.google.com by `lee.Imm...@btinternet.com` on 28 Apr 2015 at 12:45

**Comment by geoffhumphrey** [comment]
I'm assuming you mean the export all email addresses function. A club field has been added to the Email Addresses (CSV Files) > All Participants export. This will be available in the next version, 1.3.1.0.

**Comment by rgiaquinta** [comment]
Along these lines, the export of "All Entries" does not include the club affiliation column.  Would love to see this column added.
